test/performance/odp_pool_latency.c

```diff
@@ -0,0 +1,1229 @@
+	       "\n"
+	       "Usage: " PROG_NAME " [OPTIONS]\n");
+	printf("\n"
+	       "E.g. " PROG_NAME " -b %c7" DELIMITER "%c1%c" DELIMITER "%c3" DELIMITER "%c9%c\n",
+	       ALLOC, FREE, TOP, ALLOC, FREE, BOTTOM);
+	printf("     " PROG_NAME " -b %c10" DELIMITER "%c1000" DELIMITER "%c10%c -t 1 -d 2048 -p 0"
+	       " -w\n", ALLOC, DELAY, FREE, TOP);
+	printf("\n"
+	       "Mandatory OPTIONS:\n"
+	       "\n"
+	       "  -b, --burst_pattern Burst pattern for allocations, frees and delays per round,\n"
+	       "                      delimited by '%s', no spaces. Allocations are indicated\n"
+	       "                      with a '%c' prefix, frees with a '%c' prefix. The location\n"
+	       "                      of frees are indicated from the top of a previously\n"
+	       "                      allocated array of events with a '%c' suffix and from the\n"
+	       "                      bottom with a '%c' suffix. Delays are indicated with a\n"
+	       "                      '%c' prefix, followed by a delay in nanoseconds.\n"
+	       "                      Allocations and frees should be equal in the aggregate and\n"
+	       "                      frees should never outnumber allocations at any instant.\n"
+	       "                      Maximum pattern length is %u.\n"
+	       "\n"
+	       "Optional OPTIONS:\n"
+	       "\n"
+	       "  -t, --type          Pool type. %u by default.\n"
+	       "                          0: buffer\n"
+	       "                          1: packet\n"
+	       "                          2: timeout\n"
+	       "                          3: vector\n"
+	       "  -e, --event_count   Number of events. Defaults:\n"
+	       "                          buffer:  %u\n"
+	       "                          packet:  %u\n"
+	       "                          timeout: %u\n"
+	       "                          vector:  %u\n"
+	       "  -d, --data_size     Data size in bytes, ignored in case of timeout pool.\n"
+	       "                      Defaults:\n"
+	       "                          buffer: %u\n"
+	       "                          packet: %u\n"
+	       "                          vector: %u\n"
+	       "  -p, --policy        Pool allocation policy. %u by default.\n"
+	       "                      Policies:\n"
+	       "                          0: One pool shared by workers\n"
+	       "                          1: One pool per worker\n"
+	       "  -r, --round_count   Number of rounds to run. %u by default.\n"
+	       "  -i, --ignore_rounds Ignore an amount of initial rounds. %u by default.\n"
+	       "  -c, --worker_count  Amount of workers. %u by default.\n"
+	       "  -C, --cache_size    Maximum cache size for pools. Defaults:\n"
+	       "                          buffer:  %u\n"
+	       "                          packet:  %u\n"
+	       "                          timeout: %u\n"
+	       "                          vector:  %u\n"
+	       "  -w, --write_data    Write data to allocated event user areas.\n"
```

Comment:
option name could be changed write_data -> write_uarea, since it writes only user area